fdbserver/include/fdbserver/IPager.h

```diff
@@ -434,10 +471,21 @@ class ArenaPage : public ReferenceCounted<ArenaPage>, public FastAllocated<Arena
 					throw page_decoding_failed();
 				}
 			}
-			DecryptBlobCipherAes256Ctr cipher(
-			    cipherKeys.cipherTextKey, cipherKeys.cipherHeaderKey, h->encryption.iv, BlobCipherMetrics::KV_REDWOOD);
 			Arena arena;
-			StringRef plaintext = cipher.decrypt(payload, len, h->encryption, arena)->toStringRef();
+			StringRef plaintext;
+			if (CLIENT_KNOBS->ENABLE_CONFIGURABLE_ENCRYPTION) {
+				BlobCipherEncryptHeaderRef headerRef = getEncryptionHeaderRef(header);
+				const uint8_t* iv = std::visit([&](auto& algoHeader) { return algoHeader.iv; }, headerRef.algoHeader);
```

Comment:
Maybe we can use `headerRef.getIV()`?

Reply:
yeah, I had the code before `getIV()` was available and forget to update.